### PR TITLE
Add a reset view button to Icicle graphs

### LINKED ISSUE #474: Make "going back" on icicle graph clearer
Watching people use Parca today, it is clear that people don't fully understand how to go back on icicle-graphs (by clicking the root).

Some implementations of flamegraphs/icicle-graphs fade out the expanded portions, but I'm not totally convinced that that is enough, I suspect having a button to reset might be the most accessible thing, especially to newcomers.

### PR DESCRIPTION
**Pull request by yomete**
This adds a reset view button for the icicle graphs. I opted for "Reset View" as the text but let me know if you have other ideas.

This also fixes the weird behaviour with the download pprof button and the metrics tooltip. The latter would display underneath the button.

This resolves #474

**Screenshot**
image

### CODE REVIEW

File: ui/packages/app/web/src/components/MetricsGraph.tsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ export const MetricsTooltip = ({
   const highlightedNameLabel: Label.AsObject = nameLabel !== undefined ? (nameLabel) : ({ name: '', value: '' })
 
   return (
-    <div ref={setPopperElement} style={styles.popper} {...attributes.popper}>
+    <div ref={setPopperElement} style={styles.popper} {...attributes.popper} className="z-10">
```
Comment (metalmatze) [inline ui/packages/app/web/src/components/MetricsGraph.tsx:152]: ah! So the tooltip needs a higher z-index? 
Cool! Thanks for fixing this in here too!

File: ui/packages/shared/profile/src/ProfileView.tsx [hunk 2]
```diff
@@ -94,20 +95,33 @@ export const ProfileView = ({
...
               {/* TODO: Proper offset */}
-              <div className="w-full"/>
-              <div className="w-full"/>
-              <div className="w-full"/>
-              <Button color="neutral" onClick={downloadPProf}>Download pprof</Button>
+              <div className='w-1/4'>
+                <Button color='neutral' onClick={resetIcicleGraph}>
+                  Reset View
+                </Button>
+              </div>
```
[the remark this answers is not in the document]
Comment (brancz) [reply ui/packages/shared/profile/src/ProfileView.tsx:115]: Yeah I think I like the idea of disabling it until it's "useful".

**Review by brancz — APPROVED** [review]
Nice!

File: ui/packages/shared/profile/src/ProfileView.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (yomete) [reply ui/packages/shared/profile/src/ProfileView.tsx:115]: Ok then, I'll modify the button to behave that way 👍🏾

**Comment by yomete** [comment]
@brancz @metalmatze
Dec-13-2021 14-44-15

**Comment by metalmatze** [comment]
Flawless! :tada:

**Pull request merged**